modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/security/permission/resource/FragmentCollectionDLFileEntryModelResourcePermissionRegistrar.java

```diff
@@ -50,8 +52,21 @@ public void activate(BundleContext bundleContext) {
 				FragmentCollection::getFragmentCollectionId,
 				_fragmentCollectionLocalService::getFragmentCollection,
 				_portletResourcePermission,
-				(modelResourcePermission, consumer) -> {
-				}),
+				(modelResourcePermission, consumer) -> consumer.accept(
+					(permissionChecker, name, model, actionId) -> {
+						if (actionId.equals(ActionKeys.VIEW)) {
+							return true;
+						}
+
+						if (_portletResourcePermission.contains(
+								permissionChecker, model.getGroupId(),
+								FragmentActionKeys.MANAGE_FRAGMENT_ENTRIES)) {
+
+							return true;
+						}
+
+						return null;
```

Comment:
Hi @pavel-savinov, 

I'm not sure about this. Maybe we should return false here as MANAGE_FRAGMENT_ENTRIES is the only permission we need to check. Correct?